include/zephyr/net/wifi_mgmt.h

```diff
@@ -752,6 +752,10 @@ struct wifi_twt_params {
 			 * prepare the data before TWT SP starts.
 			 */
 			uint32_t twt_wake_ahead_duration;
+			/** TWT info enabled or disable */
+			bool twt_info_disable;
+			/** TWT exponent */
+			uint8_t exponent;
```

Comment:
As the (mantissa, exponent) are protocol fields, it was intentionally decided to use only the `internal` in `ms` as a user input to keep things simple and in the driver we do https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/wifi/nrf_wifi/src/wifi_mgmt.c#L250 with exponent=0. I understand that exposing both (mantissa, exponent) directly gives us flexibility but would put burden on users as they are only concerned about times.

Reply:
Under the condition exponent=0, mantissa is equal to interval, but for 'TWT Parameter Set field format' descripted in spc, mantissa use 2 Octets,  not enough to represent twt_interval, still need to set exponent value, then there will be multiple sets of exponent and mantissa, and maybe twt_interval not equal to mantissa * 2^(exponent ) accurately

Reply:
So, the last discuss point for this PR is need add exponent parameter or not.
As twt feature support by vendor's different implementation. Could we add exponent parameter, if user input exponent value from shell, then vendor use the input value, or vendor use the default value 0?

Reply:
As I said for testing all possible combinations we need both as inputs, but I am thinking about whether user's are knowledgeable enough to understand how (mantissa, exponent) translate to the wake/sleep timings. Let's see what others think.